pallets/parachain-staking/src/lib.rs

```diff
@@ -322,7 +327,8 @@ pub mod pallet {
 		/// Cancelled request to change an existing delegation.
 		CancelledDelegationRequest {
 			delegator: T::AccountId,
-			cancelled_request: DelegationRequest<T::AccountId, BalanceOf<T>>,
+			collator: T::AccountId,
+			cancelled_request: CancelledScheduledRequest<BalanceOf<T>>,
 		},
```

Comment:
The format of this event changed -- this should be mentioned in the PR description, release description.

Reply:
Also, for backward compatibility reason, new fields should be added at the end of the event